src/components/addBook.jsx

```diff
@@ -1,27 +1,45 @@
+import { nanoid } from '@reduxjs/toolkit';
 import { useDispatch } from 'react-redux';
-import { addBook } from '../redux/books/booksSlice';
+import { addBook, fetchBooks } from '../redux/books/booksSlice';
 import './addBook.scss';
 
 const AddBook = () => {
   const dispatch = useDispatch();
 
-  const handleFormSubmit = (event) => {
+  const handleFormSubmit = async (event) => {
+    dispatch(fetchBooks());
     event.preventDefault();
     const title = event.target.bookTitle.value;
     const author = event.target.author.value;
-    dispatch(addBook({ title, author }));
+    const category = event.target[2].options[event.target[2].selectedIndex].text;
+    const itemId = nanoid();
+
+    dispatch(addBook({
+      title,
+      author,
+      item_id: itemId,
+      category,
+    }));
     event.target.reset();
+    console.log(category);
   };
 
   return (
     <div className="add-book">
       <h4>ADD NEW BOOK</h4>
       <form className="row g-3 mt-2" onSubmit={handleFormSubmit}>
```

Comment:
Kindly do the following to stick to the Zeplin design:
1- It would be better to remove the inner container and use the right background
  - Your design:
![Screenshot from 2023-04-15 23-36-53](https://user-images.githubusercontent.com/97834160/232254119-c6eb1c46-73ac-4dbc-85ab-3332541318c2.png)
  - Zeplin design:
![image](https://user-images.githubusercontent.com/97834160/232254207-66edebb2-4fa8-4953-b47a-73b1e3baa192.png)